packager/packager.cc

```diff
@@ -700,15 +762,15 @@ class ImportTreeBuilder final {
                                 fmt::map_join(parts, "::", [&](const auto &nr) { return nr.show(ctx); }));
                     e.addErrorLine(core::Loc(ctx.file, parentSrc.importLoc), "Conflict from");
                 }
-            } else {
+            } else if (importType == ImportType::Test || node->source.importType == ImportType::Normal) {
```

Comment:
This codepath is now handling:

* Normal imports in normal files.
* Normal imports in test files.
* Test imports in test files.

~...but it's handling all three of these the same way. I think this is correct for the first two scenarios, but not for test files. Don't test exports live in a different `<PackageTests>` namespace?~

I got my wires crossed; this is fine. What's missing is importing `Test::` constants, which you described in the PR.

Reply:
I think this method is also the place that needs to set up importing the main package into the test package, e.g. making all of `Opus::Foo` available in `Tests::Opus::Foo`.